config/routes.rb

```diff
@@ -23,6 +23,7 @@
   get "/terms-conditions", to: "pages#terms", as: "terms"
   get "/accessibility", to: "pages#accessibility", as: "accessibility"
   get "/privacy-policy", to: "pages#privacy", as: "privacy"
+  get "/cycle_has_ended", to: "pages#cycle_has_ended", as: "cycle_has_ended"
```

Comment:
```suggestion
  get "/cycle-has-ended", to: "pages#cycle_has_ended", as: "cycle_has_ended"
```

Reply:
To avoid dropping down to using middleware, we could consider something like this:

```rb
if Settings.cycle_has_ended
  get "/cycle-has-ended", to: "pages#cycle_has_ended", as: "cycle_has_ended"
  get "/*", redirect("/cycle-has-ended")
else
  get "/cycle-has-ended", redirect("/")
end
```

Reply:
Good spot - the Rack middleware approach was redirecting all requests to the '/cycle-has-ended' page. This is wrong as we still want to maintain access to certain pages (e.g. T&Cs, Privacy Policy) as well as the healthcheck endpoint. 

Have pushed the changes